src/twisted/words/xish/xpathparser.py

```diff
@@ -43,7 +43,6 @@
 
 exec(
     r'''
-from __future__ import print_function
```

Comment:
There is this comment at the top of the file 

```
# DO NOT EDIT xpathparser.py!
```

... I guess that this is ok... we don't need to re-run Yapps for this cleanup

Reply:
I think it's fine if the two copies are updated in sync.

Reply:
No need for release notes for this.